src/riak_cs_delete_fsm.erl

```diff
@@ -240,18 +251,21 @@ maybe_delete_blocks(State=#state{bucket=Bucket,
                                     free_deleters=NewFreeDeleters,
                                     delete_blocks_remaining=NewDeleteBlocksRemaining}).
 
--spec notify_gc_worker(term(), state()) -> term().
-notify_gc_worker(Reason, State) ->
-    gen_fsm:sync_send_event(State#state.gc_worker_pid,
-                            notification_msg(Reason, State),
-                            infinity).
+-spec notify_requester(term(), state()) -> term().
+notify_requester(Reason, State = #state{finished_callback=FinishedCallback}) ->
```

Comment:
There is no notion of "requester" or "notification" in this function. More general name may be better.